app/controllers/auth/AuthAction.scala

```diff
@@ -54,8 +54,9 @@ class AuthActionImpl @Inject()(
           Retrievals.confidenceLevel and
           Retrievals.name and
           Retrievals.loginTimes and
-          Retrievals.trustedHelper) {
-        case nino ~ Enrolments(enrolments) ~ Some(credentials) ~ confidenceLevel ~ name ~ logins ~ trustedHelper =>
+          Retrievals.trustedHelper and
+          Retrievals.profile) {
+        case nino ~ Enrolments(enrolments) ~ Some(credentials) ~ confidenceLevel ~ name ~ logins ~ trustedHelper ~ profile =>
           if (trustedHelper.isDefined) {
```

Comment:
This isn't your code but consider fixing it.
Instead of checking `isDefined` use pattern matching to determine how to handle a `Some` and a `None`